public/app/plugins/panel/geomap/layers/data/markersLayer.tsx

```diff
@@ -94,24 +96,22 @@ export const markersLayer: MapLayerRegistryItem<MarkersConfig> = {
             continue; // ???
           }
 
-          const colorDim = getColorDimension(frame, style.color ?? defaultStyleConfig.color, theme);
-          const sizeDim = getScaledDimension(frame, style.size ?? defaultStyleConfig.size);
-          let textDim = undefined;
-          if (style?.text && (style.text.field || style.text.fixed)) {
-            textDim = getTextDimension(frame, style.text);
-          }
-          const opacity = style?.opacity ?? defaultStyleConfig.opacity;
 
-          const featureDimensionConfig: FeaturesStylesBuilderConfig = {
-            colorDim: colorDim,
-            sizeDim: sizeDim,
-            textDim: textDim,
-            textConfig: style?.textConfig,
-            opacity: opacity,
-            styleMaker: markerMaker,
-          };
+          if(style.fields) {
```

Comment:
```suggestion
          if (style.fields) {
```

Reply:
hymmm -- weird it let me commit without running lint.  something in precommit must have changed!